cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -93,5 +95,7 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	experimentalCmd.AddCommand(phases.NewCmdPhase(out))
 	cmds.AddCommand(experimentalCmd)
 
+	utilfeature.DefaultFeatureGate.AddFlag(cmds.PersistentFlags())
```

Comment:
@tallclair After https://github.com/kubernetes/kubernetes/pull/49015, we'd want to use the same mechanism to be consistent and make the user able to enable some of the most important server-side features via the CLI flag, but mainly add some net new items to the list that are kubeadm specific (HA for instance)